test/functional/api/buffer_updates_spec.lua

```diff
@@ -161,7 +161,7 @@ describe('API: buffer events:', function()
 
     -- create a new empty buffer and wipe out the old one ... this will
     -- turn off buffer events
-    command('enew!')
+    command('bdelete!')
```

Comment:
this doesn't match the comment. Do we need to create a new buffer also?

Reply:
I'll update the comment to read "delete the current buffer to turn off buffer events"